src/sentry/api/endpoints/team_project_index.py

```diff
@@ -112,14 +116,22 @@ def post(self, request, team):
                         name=result['name'],
                         slug=result.get('slug'),
                         organization=team.organization,
-                        team=team
+                        team=team,
+                        platform=result.get('platform')
                     )
             except IntegrityError:
                 return Response(
                     {'detail': 'A project with this slug already exists.'},
                     status=409,
                 )
 
+            platform = result.get('platform')
+            if platform:
```

Comment:
i would confirm with @ehfeng that it makes sense to capture ProjectPlatform here, given historically it has meant "has sent data with this platform", and this changes it to "has suggested any interest at all in the platform". It seems less useful if we change the meaning based on their selection here.